bindings/core/Cargo.toml

```diff
@@ -11,6 +11,7 @@ publish = false
 [dependencies]
 iota-sdk = { path = "../../sdk", default-features = false, features = [
     "wallet",
+    "samples",
```

Comment:
Is this really a good name? Not very descriptive of what it contains tbh

Reply:
suggestions?